.gitignore

```diff
@@ -13,4 +13,3 @@ venv
 dist/
 build/
 %LOCALAPPDATA%
-s2-python/src/s2python/specification/s2-pairing/s2-over-ip-pairing.yaml
```

Comment:
why did you take that out? Might be worth having if you use the yaml file for generating?